library/oneview_enclosure_facts.py

```diff
@@ -182,8 +175,6 @@ def _gather_optional_facts(self, options):
 
         ansible_facts = {}
 
-        if options.get('script'):
-            ansible_facts['enclosure_script'] = self.current_resource.get_script()
         if options.get('environmentalConfiguration'):
```

Comment:
I think these playbooks will be used for both c7000 and synergy but not sure. If this can be used for both, then it is suggested to keep the c7000 related code and add "ignore_errors" tag in the playbook while running against synergy or vice versa

Reply:
its not present in both variants from api version 500

Reply:
removed all those changes as they are only applicable only for Synergy